tensorflow/core/graph/mkl_layout_pass_test.cc

```diff
@@ -1470,7 +1720,7 @@ TEST_F(MklLayoutPassTest, NodeRewrite_QuantizeV2Op_Negative_ConstInp) {
             "A->D;B->D:1;C->D:2;D->E");
 }
 
-TEST_F(MklLayoutPassTest, NodeRewrite_QuantizeV2Op_MinFirst) {
+TEST_F(MklLayoutPassTest, NodeRewrite_QuantizeV2Op_Negative_MinFirst) {
```

Comment:
`QuantizeV2Op` seems to still support `MIN_FIRST` mode. Why do we expect this to be negative?

Reply:
Looks like I need to sync this file with latest version in master.

Reply:
Done.